src/PeriodEnd/PaymentsDue/DeployPaymentsDue/sql/dml/14 PeriodEnd.PaymentsDue.PreRun.Staging.ApprenticeshipEarnings3.sql

```diff
@@ -2,9 +2,8 @@ TRUNCATE TABLE Staging.ApprenticeshipEarnings3
 GO
```

Comment:
Nothing wrong exactly with the change, but the SQL script in general is very inefficient. The query plan shows table scans, filtering, hash matches and sorts that could be rationalised by rewriting the query.